### ISSUE TITLE
Ghostwriter collaboration (forums, slack, irc)

### ISSUE THREAD

**Issue by dbuentello** [GhostManager/Ghostwriter#39]
👋 

**TLDR;** Is there a space we can collaborate on Ghostwriter improvements and QA? For example, an IRC channel? I'd be happy to help coordinate or assist however I can. 

**Long Story**
We've (https://dragos.com/) tested Ghostwriter on several engagements and the results were great. We learned a lot and have made some tweaks specific to us. For example, we made certain sections not generate in the docx if they are empty (https://github.com/dbuentello/Ghostwriter/commit/43776edcf8798067ba7bd1e18fe8c1969249825b). Maybe others could benefit from this and I should submit a PR or maybe I'm "Ghostwriter-ing" wrong. As we start to roll this out to more users (and developers) it would be nice to have another mechanism to communicate other than Github issues.

**Comment by danms0** [comment]
Maybe the https://bloodhoundgang.herokuapp.com/ slack in a #ghostwriter channel is a good place?

**Comment by chrismaddalena** [comment]
Hey there, @dbuentello !

Issues are primarily tracked here on GitHub (and in internal tickets). Th BloodHound Slack is the best spot for live chat, as @danms0 mentioned. We could create a #ghostwriter channel if that would be of interest to folks. For the time being, we have been using the #reporting channel to discuss Ghostwriter and reporting topics.

In regards to your mentioned changes, a PR would be most welcome. Even without looking at your code I am pretty sure I know what you mean and it's been on my list of improvements to make. Q4 is just keeping me busy right now. I have some bug fixes and improvements just about ready but haven't touched that one yet.

**Comment by dbuentello** [comment]
Sweet. I was able to join Slack and the channel you mentioned. Added a PR with badge that links to Slack. I put the _#reporting_ channel but agree there should be a _#ghostwriter_ dedicated channel. Let me know if you'd like me to update PR

**Comment by mattreduce** [comment]
@chrismaddalena @dbuentello hi y'all 😄 I went ahead and made a #ghostwriter channel on BloodHound Slack so we can get into development discussions there and keep #reporting about reporting. My team at HashiCorp has started using Ghostwriter and I'd like to contribute, as well. See you there 🙂

**Comment by chrismaddalena** [comment]
@mattreduce Cool, thanks for letting us know 🙂 I will join the channel tomorrow. We have been using the #reporting channel in the BloodHound Slack. We were waiting to see if there would be a desire for  a #ghostwriter channel. Since you made one, we can use that going forward.

**Comment by chrismaddalena** [comment]
Closing this because it's not an issue. For anyone finding this, you can join the Slack here:

https://bloodhoundgang.herokuapp.com
